### DISCUSSION TITLE
posit installation

### DISCUSSION THREAD

**Discussion by AlainZuur** [JenniNiku/gllvm#173] [General]
I am not sure whether this is the right place to ask this question (maybe the posit forum is more appropriate)...but I will just try.

I wonder if anyone managed to install gllvm on posit cloud?

https://forum.posit.co/t/install-gllvm-from-cran/159858/2

Increasing memory/CPU does not help. 

Alain

**Comment by BertvanderVeen** [comment]
@JenniNiku recently had a question from someone with a similar issue, but I am not sure if she managed to resolve it.

**Comment by JenniNiku** [comment]
Hi, I did try to look into this but I didn't manage to solve it. I only tried with free version so at least for me the memory was probably the reason.

**Reply by AlainZuur** [reply]
Thanks. It is not a memory thing. If I set everything to maximum, I still get the same errors. 

I've raised a ticket with posit.cloud. If they find a solution, I will post it here for others in the future.

Alain

**Reply by AlainZuur** [reply]
Ok..here is the story.

If you try to install gllvm with the default RAM and CPU settings, it will fail. You then need to set the RAM to at least 16GB and the CPU to 4. But it will still not work, as there are some gllvm folders from the failed attempt in:

 ‘/cloud/lib/x86_64-pc-linux-gnu-library/4.3/00LOCK-gllvm’

You need to delete this 00LOCK-gllvm directory, ensure that enough RAM/CPU is selected, restart R, and then you can install it from CRAN. I assume that the development version can be installed in the same way (after restarting R).

Alain

**Reply by BertvanderVeen** [reply]
Thanks for posting the solution Alain, that is very helpful for future reference.
